tests/setuidgid/Makefile.BSD

```diff
@@ -17,6 +17,6 @@ SRCS	+=	warnp.c
 IDIRS	+=	-I${LIBCPERCIVA_DIR}/util
 
 test:	test_setuidgid
-	@./test_setuidgid.sh
+	@./test_setuidgid $$$$(id -u):$$$$(id -g)
```

Comment:
You know that this breaks `make -f Makefile.BSD test`, right?

Reply:
ok, I reverted back to using a shell script.  I'll update this PR once #112 is merged (no point in generating more `Makefile` conflicts).